vllm/model_executor/parallel_utils/communication_op.py

```diff
@@ -104,3 +109,67 @@ def broadcast_object_list(obj_list, src=0):
     # Broadcast.
     torch.distributed.broadcast_object_list(obj_list, src=src)
     return obj_list
+
+
+class TensorMetadata:
```

Comment:
Can using a class instead of raw data type incur any additional overhead? If so, can we use named tuple instead?